src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/MicrosoftCodeAnalysisExpressionEvaluator/Microsoft.CodeAnalysis.ExpressionEvaluator.swr

```diff
@@ -16,6 +16,8 @@ vs.payloads
 
 vs.dependencies
   vs.dependency id=Microsoft.VisualStudio.MinShell
+  vs.dependency id=Microsoft.Net.4.6.1.FullRedist 
```

Comment:
@Newrad0603 MinShell is there to work around a bug in the VSIXInstaller where it will remove itself before all VSIXes are uininstalled.  This does not matter for install order.